modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/ratings.js

```diff
@@ -489,9 +489,49 @@ AUI.add(
 
 						var thumbScore = instance._getThumbScores(json.totalEntries, json.totalScore);
 
+						instance._updateTitles(thumbScore);
 						instance._updateScores(thumbScore);
 					},
 
+					_updateTitles: function(thumbScore) {
+						var instance = this;
+
+						var nodes = instance.ratings.get('elements');
+
+						nodes.each(
+							function(node, i) {
+								var newTitle = '';
+
+								if ((0 === i && thumbScore.positiveVotes != node.html()) ||
+									(1 === i && thumbScore.negativeVotes != node.html())) {
```

Comment:
This makes the code too fragile. Can you try using the `rating-on` and `rating-off` CSS class isntead